_episodes/05-understanding-yaml-and-ci.md

```diff
@@ -44,7 +47,7 @@ For more details, check [this link](https://docs.github.com/en/free-pro-team@lat
 >
 > `<job_id>`: Each job must have an id to associate with the job, job_1 in the above example. The key job_id is a string that is unique to the jobs object. It must start with a letter or _ and contain only alphanumeric characters, -, or _. Its value is a map of the job's configuration data.
 >
-> - `runs-on`: **Required**. Each job runs in an virtual environment (runner) specified by the key runs-on. Available environments can be found [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on).
+> - `runs-on`: **Required**. Each job runs in a particular type of machine that we choose with this key. There are options for the major operating systems (Linux, Windows, and macOS) and different versions of them. The available options can be found [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on).
```

Comment:
These are still called "runners" in lesson 8, so it might be useful to retain this bit of jargon here.

Reply:
Okay, sounds good.